docs/application/dotnet/tutorials/certificates/creating-certificates.md

```diff
@@ -0,0 +1,116 @@
+# Creating Certificates
+
+In the menu bar, click `Tools > Tizen > Tizen Certificate Manager` to launch the Certificate Manager.
+
+## Import old profile
+
+If you have a profile you created in the past, you can import it or click a `cancel` button.
+![Migration for Certificate Profile](media/certification_guide3.png)
+
+## Create a new Certificate Profile
+
+Certificate profile will contain author certificate and distributor certificate.
+
+### 1. New Certificate profile
```

Comment:
We don't use ordered list as headings for tutorials and guides.

Reply:
Done. Please add your role on contribute.md.